packages/platforms/delta/x86-64/x86-64-delta-agc7648a/onlp/builds/src/module/auto/x86_64_delta_agc7648a.yml

```diff
@@ -0,0 +1,50 @@
+###############################################################################
+#
+# x86_64_delta_agc7648a Autogeneration Definitions.
+#
+###############################################################################
+
+cdefs: &cdefs
+- x86_64_delta_agc7648a_CONFIG_INCLUDE_LOGGING:
```

Comment:
By convention all configuration defines are upper case. 
Can you s/x86_64_delta_agc7648a_CONFIG_/X86_64_DELTA_AGC7648A_CONFIG/g?